packages/api/endpoints/token.js

```diff
@@ -28,7 +28,7 @@ function token(event, context) {
   // Code contains the value from the Earthdata Login redirect. We use it to get a token.
   if (code) {
     const params = `?grant_type=authorization_code&code=${code}&redirect_uri=${redirectUriParam()}`;
-
+    log.info("posting token verification at:", EARTHDATA_CHECK_CODE_URL + params);
```

Comment:
Is this token something sensitive that we don't want to be logging?

Reply:
Agree... removing...